src/openforms/js/components/admin/form_design/registrations/index.js

```diff
@@ -51,9 +51,12 @@ export const BACKEND_OPTIONS_FORMS = {
   },
   'stuf-zds-create-zaak:ext-utrecht': {
     uiSchema: {
-      paymentStatusUpdateXml: {
-        'ui:widget': 'textarea',
-        'ui:rows': '10',
+      paymentStatusUpdateMapping: {
```

Comment:
The resulting form field looks like this

![image](https://github.com/user-attachments/assets/d7e128c5-0ca0-4db5-a2e9-69d9b3a412a2)

Reply:
That'll do as MVP - we can clean this up in the future with a nice React-based form following the ongoing efforts for Objects API/ZGW API :) or even via the variables tab

Reply:
UI after changes to the serializer:

![image](https://github.com/user-attachments/assets/3c234ef3-e8ef-45fa-8144-77bbf7ac29ee)